content/en/blog/kernel-testing.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Kernel testing framework
+date: 2023-09-14
+author: Federico Di Pierro, Aldo Lacuku
+slug: kernel-testing-framework
+---
+
+Historically, in the Falcosecurity organization, we faced a significant challenge: there was no established method to ensure that with each iteration of our drivers, supported kernels remained unaffected. This essential process is commonly referred to as regression testing. To elaborate, we lacked a means to guarantee that a new driver release could:
+
+* Successfully compile on multiple kernel versions.
+* Pass the eBPF verifier when executed on various kernel versions.
+* Operate as expected, such as retrieving kernel events, across multiple kernel versions.
+
+To address this issue, we started a major intervention. Initially, a [proposal](https://github.com/falcosecurity/libs/blob/master/proposals/20230530-driver-kernel-testing-framework.md) was discussed and incorporated into the libs repository.
+
+Since this was a pretty novel area, there were no pre-existing tools available to tackle it. Consequently, we embarked on the development of a new completely framework.  
```

Comment:
```suggestion
Since this was a pretty novel area, there were no pre-existing tools available to tackle it. Consequently, we embarked on the development of a completely new framework.  
```